main.go

```diff
@@ -1,131 +1,136 @@
 package main
 
+// Import necessary packages and libraries
 import (
+	"fmt"
 	"log"
 	"net/http"
 	"time"
-    "fmt"
-    "os"
-    "context"
-    "sync"
-    "net/url"
-
+	"os"
+	"context"
+	"sync"
+	"net/url"
 	"github.com/gin-gonic/gin"
 	"github.com/gorilla/websocket"
 )
 
-var upgrader = websocket.Upgrader{}
-var countdown string
-var live string
-var messageOut = make(chan string)
+// Declare global variables
+var upgrader = websocket.Upgrader{} // Upgrader for WebSocket connections
+var countdown string                // String for countdown timer
+var live string                    // String for live countdown
+var messageOut = make(chan string)  // Channel for outgoing WebSocket messages
 
+// WebSocketConnection function to establish a WebSocket connection
 func WebSocketConnection() {
-    uri := url.URL{Scheme: "ws", Host: "localhost:8888", Path: "/socket",}
-    c, resp, err := websocket.DefaultDialer.Dial(uri.String(), nil);
-    if err != nil {
+	uri := url.URL{Scheme: "ws", Host: "localhost:8888", Path: "/socket"}
+	c, resp, err := websocket.DefaultDialer.Dial(uri.String(), nil)
+	if err != nil {
 		log.Printf("handshake failed %d", resp.StatusCode)
-    }
-    done := make(chan struct{})
-
-   go func() {
-   	defer close(done)
-	for {
-		_, message, err := c.ReadMessage()
-		if err != nil {
-			log.Println("read:", err)
-			return
-		}
-		messageOut <- string(message)
 	}
-   }()
-   x := <- messageOut
-   fmt.Println(x)
-   live = string(x)
+	done := make(chan struct{})
+
+	go func() {
+		defer close(done)
+		for {
+			_, message, err := c.ReadMessage()
+			if err != nil {
+				log.Println("read:", err)
+				return
+			}
+			messageOut <- string(message)
+		}
+	}()
+	x := <-messageOut
+	fmt.Println(x)
```

Comment:
Can you remove the printed variable?

Reply:
yes, I would do that just now

Reply:
> Can you remove the printed variable?

The printed variable has been removed. Let me know if you need any additional changes.

Reply:
@Gordiancyber, Great work! Have a Happy Hacktoberfest!